bin/load_images

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash -e
 
 touch .build_built
-for var in "$@"
-do
+for var in "$@"; do
+    working_dir=$PWD
```

Comment:
working_dir should go outside of for loop?  (Along with closing cd)

The redirect then can use $working_dir/.build_built

Also it's not "working_dir" -- it's the run_dir or something (since you change the working dir).

Reply:
I figured it'd be the same thing, but put it outisde the loop.